File: HLC01_02Java/src/main/java/org/example/Videojuego.java
package org.example;

public class Videojuego {
    private final String titulo;
    private final String plataforma;
    private final int horasJugadas;

    public Videojuego(String titulo, String plataforma, int horasJugadas) {
        this.titulo = titulo;
        this.plataforma = plataforma;
        this.horasJugadas = horasJugadas;
    }

    public String getTitulo() {
        return this.titulo;
    }

    @Override
    public String toString() {
        return "nombre: " + this.titulo + "\nplataforma: " + this.plataforma + "\nhoras jugadas: " + this.horasJugadas + "\n";
    }
}

File: HLC01_02Kotlin/src/main/kotlin/Videojuego.kt
class Videojuego (
    val titulo: String,
    private val plataforma: String,
    private val horasJugadas: Int
) {

    override fun toString(): String {
        return "nombre: $titulo\nplataforma: $plataforma\nhoras jugadas: $horasJugadas\n"
    }

}